phantom_token.c

```diff
@@ -246,7 +246,7 @@ static ngx_int_t handler(ngx_http_request_t *request)
 
     u_char *bearer_token_pos;
 
-    if ((bearer_token_pos = (u_char *)ngx_strcasestrn((char*)request->headers_in.authorization->value.data, BEARER,BEARER_SIZE)) == NULL)
+    if ((bearer_token_pos = ngx_strcasestrn((u_char*)request->headers_in.authorization->value.data,(char*) BEARER,BEARER_SIZE)) == NULL)
```

Comment:
the `u_char*` doesn't hurt anything I suppose, but is it needed?

```
 ngx_str_t

        A structure containing a pointer to a character array and the length of the character array

    size_t len

        The length of the character array

    unsigned char *data
```
https://www.nginx.com/resources/wiki/extending/api/utility/

Sorry to nitpick.

Reply:
Got some "note" from compiler, saying: 
src/core/ngx_string.h:163:9: note: expected ‘char *’ but argument is of type ‘const char *’
src/core/ngx_string.h:163:9: note: expected ‘u_char * {aka unsigned char *}’ but argument is of type ‘char *’
. feel annoying to have these, that's why did the casting.

Reply:
one thing I forgot to mention before, did uplift for nginx source from 1.13.3 to 1.13.5 locally

Reply:
Must just be a difference between your compiler on Linux vs. mine on Mac. It's fine to have the cast and avoid the warning (which I agree is annoying).